d3a_interface/utils.py

```diff
@@ -482,3 +484,8 @@ def sort_list_of_dicts_by_attribute(input_list: List[Dict],
         return sorted(
             input_list,
             key=lambda obj: obj.get(attribute))
+
+
+def convert_datetime_to_ui_str_format(data_time):
```

Comment:
What's the reason to implement this method on the backend side, and not on the frontend side?

Reply:
I followed the approach we have been using so far that is to pre-convert datetime to ui formatted string, for reference, have a look at similar method `convert_datetime_to_str_in_list`